plone/supermodel/schema.rst

```diff
@@ -330,7 +330,7 @@ tag to wrap a sequence of fields.
     ...             <max_length>10000</max_length>
     ...         </field>
     ...
-    ...         <fieldset name="dates" label="Dates" description="Standard dates">
+    ...         <fieldset name="dates" label="Dates" description="Standard dates" order="1">
```

Comment:
There is no order attribute, unfortunately. See: https://developer.mozilla.org/de/docs/Web/HTML/Element/fieldset
You could use ``data-order``, if necessary.